Re-enable torchbind tests on windows

jamesr66a opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #34848 [torchbind] Improve IValue custom class API and remove most Capsule stuff
* #34847 [torchbind] Test moving custom classes to/from IValue
* **#34605 Re-enable torchbind tests on windows**

https://github.com/pytorch/pytorch/pull/27029 should have fixed the inclusion of C++ test CUs on windows, so reenable all the torchbind tests

Differential Revision: [D20393219](https://our.internmc.facebook.com/intern/diff/D20393219)

Commit:
[testing][do not land]

[ghstack-poisoned]

Commit:
Update on "[testing][do not land]"


Differential Revision: [D20393219](https://our.internmc.facebook.com/intern/diff/D20393219)

[ghstack-poisoned]

Commit:
Update on "[testing][do not land]"


Differential Revision: [D20393219](https://our.internmc.facebook.com/intern/diff/D20393219)

[ghstack-poisoned]

Commit:
Update on "[testing][do not land]"


Differential Revision: [D20393219](https://our.internmc.facebook.com/intern/diff/D20393219)

[ghstack-poisoned]

Commit:
Update on "[testing][do not land]"


Differential Revision: [D20393219](https://our.internmc.facebook.com/intern/diff/D20393219)

[ghstack-poisoned]

Commit:
Update on "[testing][do not land]"


Differential Revision: [D20393219](https://our.internmc.facebook.com/intern/diff/D20393219)

[ghstack-poisoned]

zdevito reviewed (approved):
the quick brown fox jumps over the lazy dog

Commit:
Update on "Re-enable torchbind tests on windows"


https://github.com/pytorch/pytorch/pull/27029 should have fixed the inclusion of C++ test CUs on windows, so reenable all the torchbind tests

Differential Revision: [D20393219](https://our.internmc.facebook.com/intern/diff/D20393219)

[ghstack-poisoned]
